tasks/config.yml

```diff
@@ -35,6 +35,12 @@
     mode: '0600'
   notify: restart bigbluebutton
 
+- name: Encode welcome message and footer message to UTF8
+  command: native2ascii -encoding UTF8 /etc/bigbluebutton/bbb-web.properties /etc/bigbluebutton/bbb-web.properties
```

Comment:
maybe making sure that `native2ascii` is really installed would make sense.

Reply:
Do you really think this is necessary? `native2ascii` comes with `openjdk-8-jdk` and in [vars/main.yml](https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/bbb/2.3/vars/main.yml#L12) we make sure that this package is installed.